.i18nrc.json

```diff
@@ -2,6 +2,7 @@
   "paths": {
     "common.ui": "src/legacy/ui",
     "data": "src/legacy/core_plugins/data",
+    "dataNP": "src/plugins/data",
```

Comment:
What about to use `plugins_data` instead? 

@ppisljar @streamich  Do we have any rule for that?

Reply:
@azasypkin  I know you worked for i18n of Kibana for a long time. Maybe you can help us to find answer?

Reply:
Hmm, I'm not aware of any rule on that, but it'd be great if we can have one. I see we use camel cases for most of the prefixes with only `kibana_react` being an exception.

We definitely don't want to have `NP` or `LP` in names though since LP will fade away eventually and NP become a "current" platform :) But I believe we can just use `data` and have an array here like for [`xpack.spaces`](https://github.com/elastic/kibana/blob/master/x-pack/.i18nrc.json#L38), right?

Reply:
Yes, it's the best solution. Thanks for the proposal :)